File: Bootcamp_Python/time.py
# -*- coding: utf-8 -*-
"""
Created on Wed Jul  8 15:46:39 2020

@author: Cory
"""


from datetime import datetime
import time

start=time.time()
runsum=0
for x in range(100000):
    runsum+=x
print(runsum)
finish=time.time()

print("it took {} seconds to run".format(finish-start))


def getTime():
    now = datetime.now().time()
    print(now)

File: bootcamp_R_projects/time.R
Sys.time()

myTime()

start<-myTime()
runsum<-0
for(i in 1:100000){
  runsum<-runsum+i
}


finish<-myTime()
ttime=finish-start
cat("The sum was ",runsum)
cat("it took ",ttime," seconds to complete the task")



myTime <- function(){
  Sys.time()
}